mne/io/eeglab/eeglab.py

```diff
@@ -484,6 +484,8 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
             ev_idx = 0
             warn_multiple_events = False
             for ep in eeg.epoch:
+                if isinstance(ep.eventtype, int):
+                    ep.eventtype = str(ep.eventtype)
```

Comment:
@jaeilepp the `eventtype` field is like the `key` in our `event_id` dict. The only thing is that EEGLAB has it associated with every epoch